ci-operator/templates/cluster-launch-e2e-gmontero-testing.yaml

```diff
@@ -163,8 +156,6 @@ objects:
       env:
       - name: HOME
         value: /tmp/home
-      - name: TEST_COMMAND
```

Comment:
AFAICT, the only reason for having it as an environment variable before was for this test (repeated in a few templates), which would be problematic if it were done by template substitution because of quotes:

https://github.com/openshift/release/blob/081c32a31009716035ab03b03e39486cdc0940b4/ci-operator/templates/cluster-launch-e2e-upgrade.yaml#L152-L156

We don't perform that test anymore, so that's not necessary. The only place that variable appears in the templates now is here:

https://github.com/openshift/release/pull/1799/files#diff-57ef18037f2eac7815d8357fe42f0a30R160

That's a template substitution, the variable will be replaced by whatever value it has in the pod spec and quotes etc. will be preserved. I tested the instantiation with `oc process` with a few problematic values and they all worked correctly.